README.adoc

```diff
@@ -78,10 +81,16 @@ The `system` and `inventory` services can be found at the following URLs:
 
 - http://localhost:9080/inventory/systems[http://localhost:9080/inventory/systems^]
 
-Visit the http://localhost:9080/health[http://localhost:9080/health^] URL to see the health report of the two services as well as the overall
-health status of the application. One check shows the state of the `system` service, and the other shows the state of
-the `inventory` service. As you might expect, both services are in the `UP` state, and the overall health status of
-the application is `UP`.
+Visit the http://localhost:9080/health[http://localhost:9080/health^] URL to see the 
+overall health status of the application as well as the aggregated data of the liveness
+and readiness checks. Two checks show the state of the `system` service, and the other two
+show the state of the `inventory` service. As you might expect, both services are in the 
+`UP` state, and the overall health status of the application is `UP`.
```

Comment:
Change
"Visit the http://localhost:9080/health[http://localhost:9080/health^] URL to see the overall health status of the application as well as the aggregated data of the liveness and readiness checks. Two checks show the state of the `system` service, and the other two show the state of the `inventory` service. As you might expect, both services are in the `UP` state, and the overall health status of the application is `UP`."
To
"Visit the http://localhost:9080/health[http://localhost:9080/health^] URL to see the overall health status of the application, as well as the aggregated data of the liveness and readiness checks. Two checks show the state of the `system` service, and the other two checks show the state of the `inventory` service. As you might expect, both services are in the `UP` state, and the overall health status of the application is in the `UP` state."